src/perspective-mercator-viewport.js

```diff
@@ -163,15 +163,11 @@ export default class WebMercatorViewport extends Viewport {
   }
 
   getLocationAtPoint({lngLat, pos}) {
-    const c = this.project(lngLat, {topLeft: false});
```

Comment:
1) Maybe add some comment on the issues we are still seeing

2) The old code was an attempt to follow mapbox, if the new code is still shaky, should the old code be removed or just commented out? 

If you have concluded that the old code is obviously wrong and the new is obviously right, then cutting is the right choice.